testslide/matchers.py

```diff
@@ -53,6 +53,8 @@ class _AndMatcher(_AlreadyChainedMatcher):
     """
 
     def __init__(self, a: Matcher, b: Matcher) -> None:
+        if not isinstance(a, Matcher) or not isinstance(b, Matcher):
+            raise ValueError("Not of type Matcher")
```

Comment:
nit. I would have more explicit wording in here eventually pointing to what is not a `Matcher`. Like adding the presentation of `a` or `b`.
This would help developers receiving such error to be guided toward what to fix instead of having them discover it themselves

Reply:
Yes am working on it.